metrics/metricstest/metricstest.go

```diff
@@ -58,7 +58,7 @@ func CheckCountData(t *testing.T, name string, wantTags map[string]string, wantV
 		checkRowTags(t, row, name, wantTags)
 
 		if s, ok := row.Data.(*view.CountData); !ok {
-			t.Errorf("For metric %s: Reporter expected a CountData type", name)
+			t.Errorf("For metric %s: Reporter expected a CountData type, but got %T", name, row.Data)
```

Comment:
Since you're changing :)
"%s: got = %T, want:...".

Reply:
Done